project_name/local.example.env

```diff
@@ -0,0 +1,2 @@
+DJANGO_DEBUG=True
```

Comment:
Also, I think we should keep comments about these variables, on what they mean and the pattern, something like:

```
# Toggles Django Debug mode
DJANGO_DEBUG=True

# Connection database string. E.g: provider://user:password@host:port/database
DATABASE_URL=postgres://postgres:postgres@db:5432/postgres
```
I know that for these two they are kinda silly, but once we have more stuff involved, it'll be awesome!

What do you think?